cortex-mixin/dashboards/alertmanager.libsonnet

```
@@ -83,5 +87,150 @@ local utils = import 'mixin-utils/utils.libsonnet';
         $.panel('Latency') +
         utils.latencyRecordingRulePanel('cortex_request_duration_seconds', $.jobSelector($._config.job_names.gateway) + [utils.selector.re('route', 'api_v1_alerts|alertmanager')])
       )
+    )
+    .addRow(
+      $.row('Replication')
+      .addPanel(
+        $.panel('Tenants (By Instance)') +
```

[nit] Just to keep it consistent with other dasbhards.

```suggestion
        $.panel('Per %s Tenants' % $._config.per_instance_label) +
```

_Same comment applies below._